config/src/config/consensus_config.rs

```diff
@@ -6,7 +6,7 @@ use crate::config::{QuorumStoreConfig, SafetyRulesConfig};
 use serde::{Deserialize, Serialize};
 use std::path::PathBuf;
 
-pub(crate) const MAX_SENDING_BLOCK_TXNS_QUORUM_STORE_OVERRIDE: u64 = 4000;
+pub(crate) const MAX_SENDING_BLOCK_TXNS_QUORUM_STORE_OVERRIDE: u64 = 10000;
```

Comment:
I think we'll need to apply these to the "override" configs as well

```
    pub max_sending_block_bytes_quorum_store_override: u64,
    pub max_receiving_block_bytes_quorum_store_override: u64,
```

      max_sending_block_bytes: 10000000
      max_receiving_block_bytes: 15000000

Reply:
Good point - updated.